shlr/sdb/src/sdb.c

```diff
@@ -791,7 +791,7 @@ SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user) {
 	}
 
 	ut32 i;
-	for (i = 0; i < s->ht->size; ++i) {
+	for (i = 0; i < s->ht->size; i++) {
```

Comment:
This goes into the sdb repo. Not the r2 one

Reply:
ok excluded 2258c7faa95fc2c19e1fee59d3d1ffe8cc8297cd